.gitignore

```diff
@@ -40,4 +40,6 @@ scripts/.srl
 privkey.pem
 app-autoscaler-release
 docs/*.bkp
+*.drawio.bkp
+*.drawio.dtmp
```

Comment:
Do we need these patterns here? Why are these files created?

Reply:
These files come up when working with drawio app on laptop